spec/collectionspace/converter/extension/commission_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
```

Comment:
We test the addition of the extension fields once in its own spec file. (Unless we override any of the conversion logic, in which case we would test our overrides in the spec for the module doing the overrides)

Reply:
That is clear.